app/src/test/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegrationTest.kt

```diff
@@ -107,6 +108,24 @@ class BrowserToolbarIntegrationTest {
         verify(browserToolbarIntegration).observerSecurityIndicatorChanges()
     }
 
+    @Test
+    fun `WHEN start method is called THEN observe erase tabs CFR changes`() {
+
```

Comment:
nit: Extraneous empty line.
Separating the 3 main sections: arrange, act, assert is nice 👍